packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -674,3 +674,21 @@ const envoyUser = new iam.User(stack, 'envoyUser');
  */
 gateway.grantStreamAggregatedResources(envoyUser)
 ``` 
+
+## Adding Resources to shared meshes
+
+A shared mesh allows resources created by different accounts to communicate with each other in the same mesh.
+For more information about mesh sharing, see [Working with shared meshes](https://docs.aws.amazon.com/app-mesh/latest/userguide/sharing.html).
+
+```ts
+// This is the ARN for the mesh from different AWS IAM account ID.
+// Ensure mesh is properly shared with your account
+const arn = 'arn:aws:appmesh:us-east-1:123456789012:mesh/testMesh';
+appmesh.Mesh.fromMeshArn(stack, 'imported-mesh', arn);
+
+// This VirtualNode resource can communicate with the resources in the mesh from different AWS IAM account ID.
+new appmesh.VirtualNode(stack, 'test-node', {
+  mesh: sharedMesh,
```

Comment:
Why is the only example for an imported Mesh? What about a new Mesh, that's created in a CDK Stack that's in a different account than the Node?

Reply:
Adding that example.

Reply:
Actually, it would make more sense to club the two cases into creating a new mesh in CDK in a separate account, and then importing that from ARN and using it in a separate account (which is also what our example walkthrough) currently does.

Reply:
Considering this: https://github.com/aws/aws-cdk/pull/15353#discussion_r661625364, let's actually only leave the imported example (we shouldn't be showing CDK examples that require manual actions from customers between `cdk deploy` invocations).

Once we implement the functionality of sharing a Mesh automatically in a separate PR, we'll add an example for it.

Reply:
Yup, I just made it a little more descriptive.